### PR TITLE
Register Subdomain: pendrajwer

### PR DESCRIPTION
**Pull request by znawcaphotona**
Added the domain: pendrajwer

### CODE REVIEW

File: domains/pendrajwer.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "znawcaphotona",
+        "email": "wlodkowski.antek57@gmail.com"
+      },
+
+      "record": {
+        "URL": "pendrajwer.eu"
```
Comment (WilliamDavidHarrison) [inline domains/pendrajwer.json:9]: Please add more content to your website and add `https://` infront of the URL.

File: domains/pendrajwer.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+    {
+      "owner": {
+        "username": "znawcaphotona",
+        "email": "wlodkowski.antek57@gmail.com"
+      },
+
+      "record": {
+        "NS": "dns1.ct8.pl",
```
Comment (WilliamDavidHarrison) [inline domains/pendrajwer.json:9]: Sorry, we don't support NS records.

File: domains/pendrajwer.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        "NS": "dns2.ct8.pl"
```
Comment (WilliamDavidHarrison) [inline domains/pendrajwer.json:10]: Sorry, we don't support NS records.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by WilliamDavidHarrison** [comment]
Closing this PR as it is invalid and stale.

**Pull request closed without merging**